projects/eHealthSuisse/eMedikation/v1.0/1-1-MedicationTreatmentPlan.xml

```diff
@@ -187,92 +143,64 @@
 									<td ID="mtp.1.dosageevening">0</td>
 									<td ID="mtp.1.dosagenight">0</td>
 									<td ID="mtp.1.routecode">oral</td>
-									<td ID="mtp.1.note"></td>
-									<td ID="mtp.1.datefromto"></td>
+									<td ID="mtp.1.note" />
+									<td ID="mtp.1.datefromto" />
 									<td ID="mtp.1.reason">Bluthochdruck</td>
 								</tr>
 							</tbody>
 						</table>
 					</text>
-
 					<entry>
-						<substanceAdministration classCode="SBADM"
-							moodCode="INT">
-							<!-- IHE PHARM MTP Required element indicating the Medication Treatment 
-								Plan entry content module -->
+						<substanceAdministration classCode="SBADM" moodCode="INT">
+							<templateId root="2.16.756.5.30.1.1.10.4.34" />
 							<templateId root="1.3.6.1.4.1.19376.1.9.1.3.7" />
-							<!-- CCD -->
 							<templateId root="2.16.840.1.113883.10.20.1.24" />
-							<!-- PCC Medication Entry Content Module -->
 							<templateId root="1.3.6.1.4.1.19376.1.5.3.1.4.7" />
-							<!-- CH-PHARM Medication Treatment Plan Item Entry Content Module -->
-							<templateId root="2.16.756.5.30.1.1.10.4.34" />
-
-							<!-- A normal substanceAdministration act that may not contain any 
-								subordinate substanceAdministration acts. -->
 							<templateId root="1.3.6.1.4.1.19376.1.5.3.1.4.7.1" />
-
-							<!-- ID of mtp item, PCC TF2 6.3.4.16.6 -->
 							<id root="C9F758A1-296C-4710-84D4-E181DB8C7478" />
-
 							<text>
 								<reference value="#mtp.1" />
 							</text>
 							<statusCode code="completed" />
 							<effectiveTime xsi:type="IVL_TS">
 								<low value="20111129" />
+								<high nullFlavor="UNK" />
 							</effectiveTime>
-							<effectiveTime operator='A' xsi:type='EIVL_TS'>
-								<event code='ACM' />
+							<effectiveTime xsi:type="EIVL_TS" operator="A">
+								<event code="MORN" />
 							</effectiveTime>
-							<repeatNumber value="7" />
-							<routeCode code="20053000" codeSystem="0.4.0.127.0.16.1.1.2.1"
-								displayName="Oral use" />
+							<repeatNumber nullFlavor="NI" />
+							<routeCode code="20053000" codeSystem="0.4.0.127.0.16.1.1.2.1" displayName="Oral use" />

```

Comment:
This routeCode is not in the value set: https://art-decor.org/art-decor/decor-valuesets--ch-pharm-?id=2.16.756.5.30.1.1.11.2

Reply:
changed with the latest releases, will be fixed.

Reply:
Pero has added it back to the ValueSet